example_agents/papag/components.yaml

```diff
@@ -41,7 +41,7 @@ components:
         class_name: AtariEnv
         instantiate: False
 
-    CartPoleEnv==4ede9280f9c477f1ca09929d10cdc1e1ba1129f1:
+    CartPoleEnv==c755d5c35a25ab118746e2ba885894ff66fb8c43:
```

Comment:
Not sure why this line would have a colon at the end?

Reply:
This one needs a colon, right?  Because it's the key with value {repo: ..., file_path: ..., ...}

Reply:
Ah right. My bad (sorry reviewing on phone)